### PR TITLE
msg/Message.h: remove code duplication

### PR DESCRIPTION
**Pull request by stiopaa1**
Remove code duplication in constructors.

Signed-off-by: Michal Jarzabek <stiopa@gmail.com>

### CODE REVIEW

File: src/msg/Message.h [hunk 1]
```diff
@@ -221,9 +221,9 @@ class Message : public RefCountedObject {
   /* time at which message was fully read */
   utime_t recv_complete_stamp;
 
-  ConnectionRef connection;
+  ConnectionRef connection = nullptr;
```
Comment (tchaikov) [inline src/msg/Message.h]: see http://www.boost.org/doc/libs/1_61_0/libs/smart_ptr/intrusive_ptr.html#constructors

`ConnectionRef` is reset by default.

**Comment by stiopaa1** [comment]
@tchaikov 
Thanks for the links. I have updated the commit.

**Comment by tchaikov** [comment]
lgtm.

**Comment by yuriw** [comment]
http://pulpito.ceph.com/yuriw-2016-07-21_15:02:04-rados-wip-yuri-testing2_2016_7_21-distro-basic-smithi/
http://pulpito.ceph.com/yuriw-2016-07-25_09:25:30-rados-wip-yuri-testing2_2016_7_21-distro-basic-smithi/

**Pull request merged**